### ISSUE TITLE
update preview image as soon as "Count" reaches >4

### ISSUE THREAD

**Issue by JasonBuckner** [soberstadt/bill-bucks-generator#34]
Largely to address the UI issue raised in Issue #28 , it would be lovely if the preview image would change to a Vonette/Magdalena as soon as a user incremented the "Count" above 4 (even if the text on that image doesn't update until "Generate Images" is clicked).

Of course, it would also need to revert back to a Bill/Captain image when "Count" is less than 5.

**Comment by soberstadt** [comment]
Yeah, I think that is a good point as to why #28 came up. We previously had some preview functionality, but decided to ship something for #17 without adding it back in. I approve of adding it back.
